ofl/kodemono/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,10 @@
+<p>
+    A custom-designed typeface explicitly created for the developer community.
+</p>
+<p>
+    This typeface is designed to enhance the user experience and reflect our principles of functionality and
+    timelessness.
+</p>
+<p>
+    To contribute, see <a href="https://github.com/isaozler/kode-mono">github.com/isaozler/kode-mono</a>.
+</p>
```

Comment:
> WARN The last characther on DESCRIPTION.en_us.html is not a                
          line-break. Please add it. [code: missing-eof-linebreak]   

We need to let a line-break at the end of the description file, same for metadata.pb file :)

Reply:
Resolved with this commit https://github.com/google/fonts/pull/7136/commits/e753e75e8ecc8aba2feb8a84dadda50fc45ee403